java/src/main/java/io/cucumber/tagexpressions/TagExpressionParser.java

```diff
@@ -106,6 +108,7 @@ private static List<String> tokenize(String expr) {
                 isEscaped = true;
             } else if (c == '(' || c == ')' || Character.isWhitespace(c)) {
                 if (token.length() > 0) {
+                    isOperandValid(token,expr);
```

Comment:
The name of the method `isOperandValid()` is actually wrong here.
Correct, that is your intention but also operator-strings like `and`, `but` and `not` are checked by this method.

Reply:
Thank you, @jenisys  , for your review and feedback. I have made the necessary adjustment by updating the method name. As you rightly mentioned, apart from operands, this method will also handle validations for operators. Please confirm if the changes meet the requirements. Your continued assistance is appreciated

Reply:
The method-name is now OK for me 👍